templates/faq.html

```diff
@@ -49,7 +49,7 @@ <h3>Co je to PyWorking a jak se mohu přihlásit?</h3>
 
                 <h3>Mám za sebou kurz PyLadies a chci si najít první práci v IT. Co mám dělat?</h3>
                 <p>
-                    Mrkni na web <a href="https://junior.guru/">junior.guru</a>, kde najdeš tipy v tomto směru i pracovní nabídky. Autor stránek přímo spolupracuje s PyLadies. Přihlaš se <a href="http://eepurl.com/gyG8Bb">na junior.guru</a> i <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9Uh01R-uJj6B4Pyzi5rzKhlMoePTWKAhrtCqT-enaJy4rmQ/viewform?usp=sf_link">u nás na PyLadies</a> k zasílání pracovních nabídek e-mailem.
+                    Mrkni na web <a href="https://junior.guru/">junior.guru</a>, kde najdeš tipy v tomto směru i pracovní nabídky. Autor stránek přímo spolupracuje s PyLadies. <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9Uh01R-uJj6B4Pyzi5rzKhlMoePTWKAhrtCqT-enaJy4rmQ/viewform?usp=sf_link">Přihlaš se</a> k zasílání pracovních nabídek e-mailem.
```

Comment:
Currently the sentence sounds as if the job offers come from junior.guru. I suggest rewording to underline it's a form managed by PyLadies.
```suggestion
                    Mrkni na web <a href="https://junior.guru/">junior.guru</a>, kde najdeš tipy v tomto směru i pracovní nabídky. Autor stránek přímo spolupracuje s PyLadies. K zasílání pracovních nabídek e-mailem se můžeš také <a href="https://docs.google.com/forms/d/e/1FAIpQLSf9Uh01R-uJj6B4Pyzi5rzKhlMoePTWKAhrtCqT-enaJy4rmQ/viewform?usp=sf_link">přihlásit u nás</a>.
```